.gitlab-ci.yml

```diff
@@ -31,20 +30,15 @@ cache: &global_cache
   cache:
     <<: *global_cache
     policy: pull-push
-  only:
-    variables:
-      - $GITLAB_CI
-      - $CI_COMMIT_SHA
-      - $CI_COMMIT_REF_NAME
-      - $CI_JOB_ID
-      - $CI_SERVER_VERSION
-      - $CI_COMMIT_BRANCH
-      - $CI_MERGE_REQUEST_ID
-      - $PERCY_TOKEN
-      - $MYSQL_DATABASE
-      - $MYSQL_ROOT_PASSWORD
 
-before_script: []
+before_script:
+  - 'command -v ssh-agent >/dev/null || ( apt-get update -y && apt-get install openssh-client -y )'
+  - eval $(ssh-agent -s)
+  - cat $ACQUIA_PRIVATE_KEY | tr -d '\r' | ssh-add -
+  - mkdir -p ~/.ssh
+  - chmod -R 700 ~/.ssh
+  - ssh-keyscan $ACQUIA_GIT_URL >> ~/.ssh/known_hosts
+  - chmod 644 ~/.ssh/known_hosts
 
```

Comment:
From what I can tell, [Gitlab doesn't provide the same level of SSH private key support as Travis](https://docs.gitlab.com/ee/ci/ssh_keys/README.html#using-ssh-keys-with-gitlab-cicd). You basically have to manage it yourself. They have very intuitive key management for public keys that let you push _to Gitlab_, but not so much for pushing _from Gitlab to somewhere else_. I'd love a second set of eyes on this, b/c I don't love what I built here.